### PR TITLE
annotations: updated H1 2019 FF release dates, winter events, and upd…

### PR DESCRIPTION
**Pull request by SuYoungHong**
…ated indonesia holidays

### CODE REVIEW

File: usage_report/annotations/annotations_fxhealth.json [hunk 1]
```diff
@@ -77,6 +77,36 @@
         "pct_latest_version": "FF64"
       },
       "date": "2018-12-11"
+    },
+    {
+      "annotation": {
+        "MAU": "Summer Slump (Southern Hemisphere)"
```
Comment (fbertsch) [inline usage_report/annotations/annotations_fxhealth.json]: Nit: If we're putting `Southern Hemisphere` here, we should put it in the other ones for `Northern`. Generally I think we should just remove it.

**Comment by fbertsch** [comment]
Oh boy, caught up in the Test Pilot deprecation. @SuYoungHong do you have the bandwidth to fix that so the CI passes?

File: usage_report/annotations/annotations_fxhealth.json (hunk 1 above)
Comment (SuYoungHong) [reply usage_report/annotations/annotations_fxhealth.json]: okay, I'll just call it summer slump.

**Comment by SuYoungHong** [comment]
let me take a swing and see what I can do. 

You think this is the reason the job was failing last week? The test pilot URL is deprecated?

**Comment by fbertsch** [comment]
Yup, it definitely is. [We had to update some other jobs in python_mozetl](https://github.com/mozilla/python_mozetl/commit/c89fcb474f2891a83d6d0f1deb407cfbdcb3bdd9).

**Comment by SuYoungHong** [comment]
cool, talking to Teon now about how to get the replacement for `https://testpilot.firefox.com/api/experiments.json`

**Comment by SuYoungHong** [comment]
hey @fbertsch , made the following changes:
* changed `get_test_pilot_addons` function to pull from local file (which I added to the utils directory) instead of from URL (deprecated)
* removed "southern hemisphere" from the Brazil annotations. 

Hopefully tests pass and code runs now!

**Comment by fbertsch** [comment]
@SuYoungHong looks like it still isn't passing:
`IOError: [Errno 2] No such file or directory: 'experiments.json'`

**Comment by SuYoungHong** [comment]
hey @fbertsch , so fixed the filepath bug, tests are passing (but there seem to be some flake8 formatting bugs that are failing for some reason) 

did we switch python versions? it seems strange to me that these flake8 errors are popping up suddenly in parts of the file that weren't change... 

note: had to change overindented stuff in process_output.py and changed '\.' to '.' in pct_latest_version.py (i don't think removing that backslash should change anything, but would like a second opinion on it)

**Comment by SuYoungHong** [comment]
yesss! @fbertsch , tests pass now

**Comment by fbertsch** [comment]
It does seem odd. I'm not sure why it was passing earlier on those files

**Pull request merged**